src/ray/stats/metric_defs.cc

```diff
@@ -59,7 +59,8 @@ DEFINE_stats(
 DEFINE_stats(actors,
              "Current number of actors currently in a particular state.",
```

Comment:
In the long run, I wonder if it makes more sense to separate this metric cleanly rather than relying on the executor as a label. It seems tricky for users to know what the implications of source is.

Reply:
I feel if we document it correctly it should be fine? Although currently `source` is not documented at all.